### PR TITLE
Monitor

### LINKED ISSUE #18738: All the diagrams in this page are the same general AzMonitor overview visual.
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a749bf08-a45c-e441-6125-8a2bfbe7a75e
* Version Independent ID: f2fca24c-5b4e-bbf9-45d5-f7af0149ac5a
* Content: [Azure Monitor](https://docs.microsoft.com/en-us/dotnet/architecture/cloud-native/azure-monitor)
* Content Source: [docs/architecture/cloud-native/azure-monitor.md](https://github.com/dotnet/docs/blob/master/docs/architecture/cloud-native/azure-monitor.md)
* Product: **dotnet-architecture**
* Technology: **cloud-native**
* GitHub Login: @nishanil
* Microsoft Alias: **nanil**

### PR DESCRIPTION
**Pull request by robvet**
## Summary

Describe your changes here.

Fixes #18738

### CODE REVIEW

**Comment by BillWagner** [comment]
Ping @nishanil for review

**Comment by nishanil** [comment]
@sughosneo is reviewing this currently.

File: docs/architecture/cloud-native/azure-monitor.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Once the data is gathered, it can be manipulated, summarized, and plotted into c
 
 No modern application would be complete without some artificial intelligence or machine learning. To this end, data [can be passed](https://www.youtube.com/watch?v=Cuza-I1g9tw) to the various machine learning tools in Azure to allow you to extract trends and information that would otherwise be hidden.
 
-Application Insights provides a powerful query language called Kusto that can be used to find records, summarize them, and even plot charts. For instance, this query will locate all the records for the month of November 2007, group them by state, and plot the top 10 as a pie chart.
+Application Insights provides a powerful (SQL-Like) query language called *Kusto* that can find records, summarize them, and even plot charts. For example, the following query will locate all records for the month of November 2007, group them by state, and plot the top 10 as a pie chart.
```
Comment (IEvangelist) [inline docs/architecture/cloud-native/azure-monitor.md:28]:
```suggestion
Application Insights provides a powerful (SQL-like) query language called *Kusto* that can query records, summarize them, and even plot charts. For example, the following query will locate all records for the month of November 2007, group them by state, and plot the top 10 as a pie chart.
```

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready. Minor nits, otherwise this looks great!

File: docs/architecture/cloud-native/azure-monitor.md (hunk 1 above)
Comment (robvet) [reply docs/architecture/cloud-native/azure-monitor.md:28]: Good catch.

**Comment by nishanil** [comment]
@robvet  PowerBI dashboard image has a few email addresses. Is that a publicly listed image?

**Comment by nishanil** [comment]
Monitor
 Can you use this one, please?

**Review by nishanil — APPROVED** [review]

**Pull request merged**